pages/gnosis/gnosis.md

```diff
@@ -0,0 +1,41 @@
+---
+name: "Gnosis"
+logo: "gnosis.svg"
+color: "#f3f2f2"
+schemas:
+  - "gnosis"
+links:
+  - name: "Website"
+    link: "https://gnosis.io/"
+  - name: "Gnosis Safe"
+    link: "https://gnosis-safe.io/"
+  - name: "Gnosis Protocol"
+    link: "https://docs.gnosis.io/protocol/"
+  - name: "Github"
+    link: "https://github.com/gnosis"
+tags:
+  - "Gnosis Safe"
+  - "Gnosis Protocol"
+  - "DEX"
+  - "DeFi"
+  - "IDO"
+dashboards:
+  - "gnosis-safe_2"
+  - "HtS7kMiZWYS2bxA5oArxlevFwX4B0O3ZZAiRxcd3"
```

Comment:
This doesn't look like it is a dashboard

Reply:
It's supposed to be https://explore.duneanalytics.com/public/dashboards/HtS7kMiZWYS2bxA5oArxlevFwX4B0O3ZZAiRxcd3

I think it should also be available under https://explore.duneanalytics.com/dashboard/kpi-gnosis-protocol but when I try to access it via that link I only see "This dashboard is private"

Reply:
Ah got it! reached out on TG.